NitroxServer/Server.cs

```diff
@@ -52,7 +52,14 @@ public Server(WorldPersistence worldPersistence, World world, ServerConfig serve
             saveTimer.AutoReset = true;
             saveTimer.Elapsed += delegate
             {
-                Save();
+                if (!serverConfig.DisableAutoBackup)
+                {
+                    BackUp();
```

Comment:
I see that this method calls `Save()` after a conditional check. Why don't we just call `Save()` here either way? Is there a significant behavior change when checking `IsRunning`?

Reply:
The `BackUp();` function calls `Save();`, so doing that would mean saving the world twice (leading to log spam)